broker-cli/proto/broker.proto

```diff
@@ -931,6 +931,23 @@ message ReleaseChannelsRequest {
   bool force = 2;
 }
 
+/**
+ * Container for channel information
+ */
+message ReleasedChannel {
```

Comment:
calling this `ReleasedChannel` is a little weird since it could represent more than one channel.

Reply:
@treygriffith What do you mean. This message type represents a single channel